exchange/exchange.go

```diff
@@ -1180,6 +1180,9 @@ func (e *exchange) makeExtBidResponse(adapterBids map[openrtb_ext.BidderName]*en
 		}
 		if len(errList) > 0 {
 			bidResponseExt.Errors[openrtb_ext.PrebidExtKey] = errsToBidderErrors(errList)
+			if prebidWarn := errsToBidderWarnings(errList); len(prebidWarn) > 0 {
+				bidResponseExt.Warnings[openrtb_ext.PrebidExtKey] = errsToBidderWarnings(errList)
```

Comment:
Can't `bidResponseExt.Warnings[openrtb_ext.PrebidExtKey]` be set to `prebidWarn`, instead of calling `errsToBidderWarnings` again?

Reply:
Done